requirements.txt

```diff
@@ -18,3 +18,5 @@ pbkdf2
 scrypt
 pexpect
 pyelliptic==1.5.7
+mock==2.0.0
+pytest-mock==1.6.0
```

Comment:
It would be better to keep this dev dependencies in a seperate file, like what pyethereum does.